web/src/Loader.css

```diff
@@ -0,0 +1,56 @@
+.loader-wrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 1000;
+}
+
+.loader {
+  display: block;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  width: 150px;
+  height: 150px;
+  margin: -75px 0 0 -75px;
+  border: 3px solid transparent;
+  border-top: 3px solid #4FB5FF;
+  border-radius: 50%;
+  z-index: 1500;
+  animation: spin 1.5s linear infinite;
```

Comment:
Duplicate `animation` - can you check whether it'll work if only one of these is present in `.loader` and `.loader:before`?

Reply:
it works, all done :)